[2025-06-08 17:51:09,378: INFO: main: This is an info message from the Kidney Disease Classification module.]
[2025-06-08 18:09:17,141: INFO: main: This is an info message from the Kidney Disease Classification module.]
[2025-06-08 18:16:22,284: INFO: main: This is an info message from the Kidney Disease Classification module.]
[2025-06-21 10:37:51,950: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-06-21 10:37:51,964: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-21 10:37:51,978: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-21 10:37:51,979: INFO: common: created directory at: artifacts]
[2025-06-21 10:37:51,979: INFO: common: created directory at: artifacts/data_ingestion]
[2025-06-21 10:37:51,979: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/11-1L5Ee7Ot-oDBnJY-B6CDwgnM8rFC6e/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-06-21 10:41:08,670: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/11-1L5Ee7Ot-oDBnJY-B6CDwgnM8rFC6e/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-06-21 10:41:51,667: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-06-23 11:09:50,803: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 11:16:34,543: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 11:16:34,578: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-23 11:17:47,533: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 11:17:47,538: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-23 11:17:47,543: INFO: common: created directory at: artifacts]
[2025-06-23 11:17:47,545: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-06-23 11:19:34,998: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 11:19:35,002: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-23 11:19:35,004: INFO: common: created directory at: artifacts]
[2025-06-23 11:19:35,006: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-06-23 11:21:15,443: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 11:21:15,448: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-23 11:21:15,450: INFO: common: created directory at: artifacts]
[2025-06-23 11:21:15,452: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-06-23 11:23:31,714: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 11:23:31,718: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-23 11:23:31,720: INFO: common: created directory at: artifacts]
[2025-06-23 11:23:31,722: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-06-23 11:23:32,218: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-06-23 12:39:24,463: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-06-23 12:39:24,467: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 12:39:24,474: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-23 12:39:24,475: INFO: common: created directory at: artifacts]
[2025-06-23 12:39:24,476: INFO: common: created directory at: artifacts/data_ingestion]
[2025-06-23 12:39:24,476: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/11-1L5Ee7Ot-oDBnJY-B6CDwgnM8rFC6e/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-06-23 12:45:13,239: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/11-1L5Ee7Ot-oDBnJY-B6CDwgnM8rFC6e/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-06-23 12:45:32,892: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-06-23 12:45:32,893: INFO: main: *******************]
[2025-06-23 12:45:32,894: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2025-06-23 12:45:32,897: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 12:45:32,899: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-23 12:45:32,900: INFO: common: created directory at: artifacts]
[2025-06-23 12:45:32,901: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-06-23 12:45:32,901: ERROR: main: get_base_model() missing 1 required positional argument: 'self']
Traceback (most recent call last):
  File "main.py", line 22, in <module>
    prepare_base_model.main()
  File "C:\Users\Chirag Sharma\Desktop\gitdemo\Kidney-Disease-Classification\src\Kidney_Disease_Classification\pipeline\stage_02_prepare_base_model.py", line 16, in main
    PrepareBaseModel.get_base_model()
TypeError: get_base_model() missing 1 required positional argument: 'self'
[2025-06-23 12:49:05,789: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-06-23 12:49:05,791: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 12:49:05,794: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-23 12:49:05,795: INFO: common: created directory at: artifacts]
[2025-06-23 12:49:05,796: INFO: common: created directory at: artifacts/data_ingestion]
[2025-06-23 12:49:05,796: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/11-1L5Ee7Ot-oDBnJY-B6CDwgnM8rFC6e/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-06-23 12:56:18,745: ERROR: main: ("Connection broken: ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None)", ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None))]
Traceback (most recent call last):
  File "C:\Users\Chirag Sharma\anaconda3\envs\Kidney\lib\site-packages\urllib3\response.py", line 748, in _error_catcher
    yield
  File "C:\Users\Chirag Sharma\anaconda3\envs\Kidney\lib\site-packages\urllib3\response.py", line 873, in _raw_read
    data = self._fp_read(amt, read1=read1) if not fp_closed else b""
  File "C:\Users\Chirag Sharma\anaconda3\envs\Kidney\lib\site-packages\urllib3\response.py", line 856, in _fp_read
    return self._fp.read(amt) if amt is not None else self._fp.read()
  File "C:\Users\Chirag Sharma\anaconda3\envs\Kidney\lib\http\client.py", line 459, in read
    n = self.readinto(b)
  File "C:\Users\Chirag Sharma\anaconda3\envs\Kidney\lib\http\client.py", line 503, in readinto
    n = self.fp.readinto(b)
  File "C:\Users\Chirag Sharma\anaconda3\envs\Kidney\lib\socket.py", line 681, in readinto
    return self._sock.recv_into(b)
  File "C:\Users\Chirag Sharma\anaconda3\envs\Kidney\lib\ssl.py", line 1274, in recv_into
    return self.read(nbytes, buffer)
  File "C:\Users\Chirag Sharma\anaconda3\envs\Kidney\lib\ssl.py", line 1132, in read
    return self._sslobj.read(len, buffer)
ConnectionResetError: [WinError 10054] An existing connection was forcibly closed by the remote host

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\Chirag Sharma\anaconda3\envs\Kidney\lib\site-packages\requests\models.py", line 820, in generate
    yield from self.raw.stream(chunk_size, decode_content=True)
  File "C:\Users\Chirag Sharma\anaconda3\envs\Kidney\lib\site-packages\urllib3\response.py", line 1060, in stream
    data = self.read(amt=amt, decode_content=decode_content)
  File "C:\Users\Chirag Sharma\anaconda3\envs\Kidney\lib\site-packages\urllib3\response.py", line 949, in read
    data = self._raw_read(amt)
  File "C:\Users\Chirag Sharma\anaconda3\envs\Kidney\lib\site-packages\urllib3\response.py", line 902, in _raw_read
    self._fp.close()
  File "C:\Users\Chirag Sharma\anaconda3\envs\Kidney\lib\contextlib.py", line 131, in __exit__
    self.gen.throw(type, value, traceback)
  File "C:\Users\Chirag Sharma\anaconda3\envs\Kidney\lib\site-packages\urllib3\response.py", line 775, in _error_catcher
    raise ProtocolError(f"Connection broken: {e!r}", e) from e
urllib3.exceptions.ProtocolError: ("Connection broken: ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None)", ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "main.py", line 11, in <module>
    obj.main()
  File "C:\Users\Chirag Sharma\Desktop\gitdemo\Kidney-Disease-Classification\src\Kidney_Disease_Classification\pipeline\stage_01_data_ingestion.py", line 16, in main
    data_ingestion.download_file()
  File "C:\Users\Chirag Sharma\Desktop\gitdemo\Kidney-Disease-Classification\src\Kidney_Disease_Classification\components\data_ingestion.py", line 33, in download_file
    raise e
  File "C:\Users\Chirag Sharma\Desktop\gitdemo\Kidney-Disease-Classification\src\Kidney_Disease_Classification\components\data_ingestion.py", line 28, in download_file
    gdown.download(prefix+file_id,zip_download_dir)
  File "C:\Users\Chirag Sharma\anaconda3\envs\Kidney\lib\site-packages\gdown\download.py", line 368, in download
    for chunk in res.iter_content(chunk_size=CHUNK_SIZE):
  File "C:\Users\Chirag Sharma\anaconda3\envs\Kidney\lib\site-packages\requests\models.py", line 822, in generate
    raise ChunkedEncodingError(e)
requests.exceptions.ChunkedEncodingError: ("Connection broken: ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None)", ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None))
[2025-06-23 13:14:18,537: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-06-23 13:14:18,537: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 13:14:18,546: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-23 13:14:18,546: INFO: common: created directory at: artifacts]
[2025-06-23 13:14:18,546: INFO: common: created directory at: artifacts/data_ingestion]
[2025-06-23 13:14:18,546: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/11-1L5Ee7Ot-oDBnJY-B6CDwgnM8rFC6e/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-06-23 13:18:42,262: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/11-1L5Ee7Ot-oDBnJY-B6CDwgnM8rFC6e/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-06-23 13:19:09,094: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-06-23 13:19:09,094: INFO: main: *******************]
[2025-06-23 13:19:09,094: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2025-06-23 13:19:09,094: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 13:19:09,094: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-23 13:19:09,094: INFO: common: created directory at: artifacts]
[2025-06-23 13:19:09,103: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-06-23 13:19:10,228: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-06-23 13:19:12,119: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-06-23 13:34:44,664: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 13:34:44,677: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-23 13:34:44,677: INFO: common: created directory at: artifacts]
[2025-06-23 13:43:46,238: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 13:43:46,248: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-23 13:43:46,256: INFO: common: created directory at: artifacts]
[2025-06-23 13:45:01,316: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 13:45:01,320: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-23 13:45:01,323: INFO: common: created directory at: artifacts]
[2025-06-23 13:45:26,706: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 13:45:26,708: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-23 13:45:26,716: INFO: common: created directory at: artifacts]
[2025-06-23 13:45:26,725: INFO: common: created directory at: artifacts\training]
[2025-06-23 13:47:36,147: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 13:49:30,797: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 13:49:30,809: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-23 13:49:30,813: INFO: common: created directory at: artifacts]
[2025-06-23 13:49:30,817: INFO: common: created directory at: artifacts\training]
[2025-06-23 13:51:14,584: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 13:51:14,627: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-23 13:51:14,632: INFO: common: created directory at: artifacts]
[2025-06-23 13:51:14,637: INFO: common: created directory at: artifacts\training]
[2025-06-23 14:01:53,833: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 14:01:53,850: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-23 14:01:53,859: INFO: common: created directory at: artifacts]
[2025-06-23 14:01:53,867: INFO: common: created directory at: artifacts\training]
[2025-06-23 14:03:53,658: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 14:03:53,665: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-23 14:03:53,668: INFO: common: created directory at: artifacts]
[2025-06-23 14:03:53,672: INFO: common: created directory at: artifacts\training]
[2025-06-23 14:04:58,903: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 14:04:58,910: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-23 14:04:58,915: INFO: common: created directory at: artifacts]
[2025-06-23 14:04:58,918: INFO: common: created directory at: artifacts\training]
[2025-06-23 14:06:31,429: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 14:06:31,436: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-23 14:06:31,439: INFO: common: created directory at: artifacts]
[2025-06-23 14:06:31,442: INFO: common: created directory at: artifacts\training]
[2025-06-23 14:12:23,335: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 14:12:23,357: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-23 14:12:23,367: INFO: common: created directory at: artifacts]
[2025-06-23 14:12:23,378: INFO: common: created directory at: artifacts\training]
[2025-06-23 14:15:37,286: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 14:15:37,294: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-23 14:15:37,298: INFO: common: created directory at: artifacts]
[2025-06-23 14:15:37,302: INFO: common: created directory at: artifacts\training]
[2025-06-23 14:33:23,242: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 14:33:23,262: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-23 14:33:23,269: INFO: common: created directory at: artifacts]
[2025-06-23 14:33:23,278: INFO: common: created directory at: artifacts\training]
[2025-06-23 15:41:40,364: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 15:41:40,379: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-23 15:41:40,387: INFO: common: created directory at: artifacts]
[2025-06-23 15:41:40,400: INFO: common: created directory at: artifacts\training]
[2025-06-23 16:24:40,722: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 16:24:40,755: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-23 16:24:40,768: INFO: common: created directory at: artifacts]
[2025-06-23 16:24:40,793: INFO: common: created directory at: artifacts\training]
[2025-06-23 16:41:51,432: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-06-23 16:41:51,462: INFO: common: yaml file: params.yaml loaded successfully]
[2025-06-23 16:41:51,481: INFO: common: created directory at: artifacts]
[2025-06-23 16:41:51,532: INFO: common: created directory at: artifacts\training]
